gcp_variant_transforms/options/variant_transform_options_test.py

```diff
@@ -93,7 +93,8 @@ def _make_args(self, args):
 
   def test_valid_table_path(self):
     args = self._make_args(['--append',
-                            '--output_table', 'project:dataset.table'])
+                            '--output_table', 'project:dataset.table',
```

Comment:
I think we should improve this test and verify somewhere that the code actually works - for example, we can add a test to catch an error when "<table_id>_call_info" actually exists.